ts/features/wallet/onboarding/bancomat/screens/add-pans/AddBancomatComponent.tsx

```diff
@@ -39,6 +43,10 @@ const styles = StyleSheet.create({
   title: { lineHeight: 33, alignSelf: "flex-start" },
   flexStart: { alignSelf: "flex-start" }
 });
+
+const isBancomatBlocked = (pan: Card) =>
```

Comment:
Could we place this function in `utils/paymentMethod.ts`?
Since it could be used from different parts

Reply:
Yep sure!